source/cloud/aws/sagemaker.md

```diff
@@ -45,7 +45,7 @@ sudo -u ec2-user -i <<'EOF'
 mamba create -y -n rapids {{ rapids_conda_channels }} {{ rapids_conda_packages }} \
     boto3 \
     ipykernel \
-    sagemaker
+    'sagemaker>=2.198'
```

Comment:
```suggestion
    sagemaker>=2.198
```

Blegh, sorry, this needs to be removed. Adding it DOES technically allow `import sagemaker` to work, but for a terrible reason... the `rapids` conda environment is just never created, causing the `rapids` kernel to not contain any of its own packages.

Running the following in a notebook using that empty kernel...

```python
import sagemaker
print(sagemaker.__file__)
```

Suggests that the import is finding the globally installed copy of that package:

```text
/home/ec2-user/anaconda3/lib/python3.10/site-packages/sagemaker/__init__.py
```

So this is really fully broken for RAPIDS 24.10 until #443 is fixed.

Reply:
I've updated the PR title and description accordingly.